src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -206,62 +444,104 @@ export default function Transaction({ hash: id }: { hash: string }) {
 
   const { activity, status, type } = activityData
 
+  // Type of Statuses
   const isPending = status === ActivityStatus.PENDING
   const isConfirmed = status === ActivityStatus.CONFIRMED
   const isExpired = status === ActivityStatus.EXPIRED
   const isCancelling = status === ActivityStatus.CANCELLING
   const isCancelled = status === ActivityStatus.CANCELLED
   const isCancellable = isPending && type === ActivityType.ORDER
 
+  // Type of Transaction
+  const isTransaction = type === ActivityType.TX
+
+  // Flags
+  const isPriceOutOfRange = false
+
   const onCancelClick = () => setShowCancelModal(true)
   const onDismiss = () => setShowCancelModal(false)
 
+  // const isLoading = isPending || isCancelling ? true : false
+
   return (
-    <RowWrapper>
-      <TransactionState href={getEtherscanLink(chainId, id, 'transaction')}>
-        <RowFixed>
-          {activity && (
-            <Pill color="#fff" bgColor={determinePillColour(status, type)} minWidth="3.5rem">
-              {type}
-            </Pill>
-          )}
-          <TransactionStatusText>
-            {isCancelling ? (
-              <MouseoverTooltip text={activity.summary || id}>
-                {getActivitySummary({ activityData, id, suffix: '(Cancellation requested)' })}
-              </MouseoverTooltip>
+    <Wrapper>
+      <TransactionWrapper>
+        <TransactionState href={getEtherscanLink(chainId, id, 'transaction')}>
+          <RowFixed>
+            {activity && (
+              <IconType color={determinePillColour(status, type)}>
+                <IconWrapper pending={isPending || isCancelling} success={isConfirmed || isCancelled}>
+                  {isPending || isCancelling ? (
+                    <Loader />
+                  ) : isConfirmed ? (
+                    <SVG src={TxCheckImage} description="Order Filled" />
+                  ) : isExpired ? (
+                    <SVG src={TxArrowsImage} description="Order Expired" />
+                  ) : isCancelled ? (
+                    <SVG src={TxArrowsImage} description="Order Cancelled" />
+                  ) : (
+                    <SVG src={TxArrowsImage} description="No state" />
+                  )}
+                </IconWrapper>
+              </IconType>
+            )}
+            <TransactionStatusText>
+              {isCancelling ? (
+                <MouseoverTooltip text={activity.summary || id}>
+                  {getActivitySummary({ activityData, id })}
+                </MouseoverTooltip>
+              ) : (
+                getActivitySummary({ activityData, id })
+              )}
```

Comment:
Since this has been redesigned, you can probably get rid of the tooltip for orders being cancelled